sdks/java/harness/src/main/java/org/apache/beam/fn/harness/data/BeamFnDataGrpcClient.java

```diff
@@ -76,7 +76,7 @@ public BeamFnDataGrpcClient(
    * (signaled by an empty data block), the returned future is completed successfully.
    */
   @Override
-  public <T> CompletableFuture<Void> forInboundConsumer(
+  public <T> CompletableFuture<Void> receive(
```

Comment:
This implies that things are received/sent but in reality things are constructed or cached objects are returned allowing one to receive/send. I think forInbound/forOutbound (w/o consumer) still works better then receive/send but I'll let you make the final decision on this.